PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.h

```diff
@@ -175,7 +203,7 @@ class AliAnalysisTaskEmcalJetPerformance : public AliAnalysisTaskEmcalJet {
   AliAnalysisTaskEmcalJetPerformance &operator=(const AliAnalysisTaskEmcalJetPerformance&); // not implemented
 
   /// \cond CLASSIMP
-  ClassDef(AliAnalysisTaskEmcalJetPerformance, 4);
+  ClassDef(AliAnalysisTaskEmcalJetPerformance, 6);
   /// \endcond
```

Comment:
Did you mean to increment by 2?

Reply:
Hey @raymondEhlers, 

It is just because there are two commits in the pull request and in each one I incremented the ClassDef.

Cheers,
James

Reply:
Ahh, I see - you're right (although practically speaking, I don't think anyone is super likely to check out between your commits, so you could probably just increment by one).